lib/reference/selectors.mjs

```diff
@@ -231,6 +255,15 @@ export const linguisticPseudoClasses = new Set(['dir', 'lang']);
 
 export const logicalCombinationsPseudoClasses = new Set(['has', 'is', 'matches', 'not', 'where']);
 
+export const deprecatedPseudoClasses = new Set([
```

Comment:
note

Depending on whether the PR that adds `selector-no-deprecated` will have a fixer or not, `deprecatedPseudoClasses` should or should not be exported.